### ISSUE TITLE
simple storage crashes if you craft too fast

### ISSUE THREAD

**Issue by crahstringray** [Darkosto/SkyFactory4Issues#475]
## Steps to Reproduce (for bugs)

1.
2.

## Logs

* Client/Server Log:
* Crash Log:

## World Information

* Preset:

* Prestige: 

* Modpack Version world created in:

* Additional Content Installed:

## Client Information

* Modpack Version:
* Java Version:
* Launcher Used:
* Memory Allocated:
* Server/LAN/Single Player:
* Optifine Installed:
* Shaders Enabled:

## Server Information
* Java Version:
* Operating System:
* Hoster/Hosting Solution:
* Sponge (Non-Vanilla Forge) Server:

**Comment by DarkBreaker09** [comment]
Got the same Problem today. 
I mistakenly used shift while Crafting via the Simple Storage Remote tool.

My Game froze for a moment like with Refined Storage when you craft to 
fast but after 2 mins or so it Crashed without creating an Error Log.

**Comment by kulwych** [comment]
agreed on this. when crafting essence, it will get some of a stack crafted, crash the game. when reloading the game, the rest is crafted.  Main issue being that every time i try to craft a stack with this mod, it crashes the game.

**Comment by DarkBreaker09** [comment]
I did a bit of testing since my Post here, its not happening with all stuff.
 
Slime and Amber for an example works fine because you only craft One stack at a time.

But if you use Logs to Craft Sticks for example it can Craft more than one Stack at a time, 
thats when the Game crasshes than.

In my Expirience at least. So i did watch what and how many i Craft at a time and the Crashes stoped.

Maybe its just that it produce too much data that Java cant handle and force the Crash, but
I am not an expert with that kind of stuff. Someone with more Expirience should look into it.
